libs/service-portal/settings/personal-information/src/components/Forms/PhoneForm/Steps/FormStep.tsx

```diff
@@ -78,6 +78,7 @@ export const FormStep: FC<Props> = ({
               hasError={errors.tel}
               errorMessage={errors.tel?.message}
               onChange={onChange}
+              maxLength={7}
```

Comment:
We don't allow spaces or dashes? E.g. `123-4567` or `123 4567` and strip out dash/space?

Reply:
We do not, but come to think of this now, this should replicate the behaviour of the log-in 2factor screen where there is a space after the first three characters.